### DISCUSSION TITLE
2022-08-18 Community Call

### DISCUSSION THREAD

**Discussion by gonuke** [svalinn/DAGMC#826] [Community Calls]
8 AM CDT/1300 UTC on Thursday, August 18.

Present: 
* UW-Madison: @gonuke, @bohmt 
* ANL: @pshriwise 
* UKAEA: @makeclean 
* First Light Fusion: @shimwell 
* BaM Scientific Consulting: @bam241 

Discussion:
* MOAB install challenges with newer version of `pip/setuptools` - fails to install testing version causing other builds to fail
    * solution - add install location to `PYTHONPATH` in moab/pymoab `CMakeLists.txt` - @gonuke 
* Assistance to Oxford Sigma: MOAB cub-file reader is falling behind Cubit. This is used in workflows that generate tetmesh for use in DAGMC tallies.  One option is to export a different mesh format from Cubit (exodus?) and configure MOAB to read that.
* Cubit plugin updates: 
    * [PR#135](https://github.com/svalinn/Cubit-plugin/pull/135) v2022.6 is now removed from Cubit so roll this back to 2022.4
    * don't worry about windows builds for now
    * add a table to Readme that shows which releases work for which configurations ✔️ see [PR](https://github.com/svalinn/Cubit-plugin/pull/137)
* Coreform still interested in pursuing OFES support
* @shimwell is working on methods of creating DAGMC geometry from vertices and tris [:link: to repo](https://github.com/fusion-energy/vertices_to_h5m)
* PyNE bug on testing with outdated OpenMC statepoint file [(Issue#1458)](https://github.com/pyne/pyne/issues/1458)

**Comment by gonuke** [comment]
Reminder to @makeclean @shimwell @pshriwise @bam241 that we are moving to use the GH Discussion threads for our meeting agenda & minutes...

**Reply by pshriwise** [reply]
Thanks! I definitely put a note in that earlier today. 🧓🏻 🐶

**Comment by pshriwise** [comment]
Discussion point: Had a call with Matt Sederberg at Coreform about a proposal to FES.

**Comment by shimwell** [comment]
I would be keen to cover thiese issues / PRs

DAGMC PRs
Adding safe folder to allow CI to compile DAGMC https://github.com/svalinn/DAGMC/pull/816
Putting install scripts into dockerfile https://github.com/svalinn/DAGMC/pull/822

PyNe issues
OpenMC testing statepoint file has wrong version https://github.com/pyne/pyne/issues/1458
